SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -136,4 +137,9 @@ mount_overlayfs() {
 }
 
 parse_kernel_cmdline_args
-mount_overlayfs
+
+if [ -n "${overlayfs}" ]; then
```

Comment:
You may want to add a comment explaining the logic here.
Looks like parse_kernel_cmdline_args should never fail and you rely on checking the variable to check if parsing was successful or not.

Reply:
Fixed, thanks.